.github/workflows/linux.yml

```
@@ -166,16 +166,34 @@ jobs:
     - name: Clear CCache stats
       run: ccache --show-stats --zero-stats
 
+    - name: Set /proc/sys/kernel/core_pattern
+      if: ${{ matrix.build-type == 'Debug' }} || ${{ matrix.build-type == 'Coverage' }}
```

It seems this is running in [Release](https://github.com/canonical/multipass/actions/runs/13585828619/job/37980523777?pr=3958) and [Clang](https://github.com/canonical/multipass/actions/runs/13585828619/job/37980523465?pr=3958), I think it might have to do with having two separate expressions. 
```suggestion
      if: ${{ matrix.build-type == 'Debug' || matrix.build-type == 'Coverage' }}
```

---

Ah, good catch. It's still weird that it causes the pipeline to run for those, though.

---

Made the change; let's see if it works as intended.

---

Yep, that was it. The step is only running for debug & coverage now.